src/consumer.ts

```diff
@@ -118,7 +129,7 @@ export class Consumer extends EventEmitter {
   private pollingWaitTimeMs: number;
   private terminateVisibilityTimeout: boolean;
   private heartbeatInterval: number;
-  private sqs: SQS;
+  private sqs: SQSClient;
```

Comment:
I think we should use the SQS interface instead of the SQSClient interface.
That way we don't need to call sqs.send but can reuse the aws-sdk's v2 method name

Reply:
I chose the v3 style because [the docs state](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-sqs/index.html)

> The client can also send requests using v2 compatible style. However, it results in a bigger bundle size and may be dropped in next major version. More details in the blog post on [modular packages in AWS SDK for JavaScript](https://aws.amazon.com/blogs/developer/modular-packages-in-aws-sdk-for-javascript/)

I don't really have any preference, my involvement with this library was extremely ephemeral